src/bounded_bls12381.rs

```diff
@@ -0,0 +1,183 @@
+/*
+ * Copyright 2020 - MATTR Limited
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+use crate::{gen_signature_message, prelude::BlsKeyPair, utils::set_panic_hook, BbsVerifyResponse};
+
+use bbs::prelude::*;
+use serde::{Deserialize, Serialize};
+use std::{
+    collections::{BTreeMap, BTreeSet},
+    convert::TryInto,
+};
+use wasm_bindgen::prelude::*;
+
+wasm_impl!(
+    BoundedBlsSignatureRequestContextRequest,
+    issuerPublicKey: Vec<u8>,
```

Comment:
```suggestion
    issuerPublicKey: DeterministicPublicKey,
```
`DeterministicPublicKey` is more appropriate for the type of issuer public key here, which I think is consistent with other interfaces.

Reply:
I absolutely agree with you.